src/app/core/entity/schema/default-value-config.ts

```diff
@@ -38,8 +38,32 @@ export type DefaultValueMode =
   | "dynamic"
   | "AutomatedConfigRule";
 
+/**
+ * Special default value rule to continuously updates the value based a field of another related entity.
+ */
 export interface AutomatedConfigRule {
-  relatedEntity: string;
-  relatedField: string;
+  /**
+   * ID of the related entity that triggers updates of this value
+   */
+  relatedEntityId: string;
```

Comment:
setting up a sample config, I now believe this is wrongly labeled:
Actually this holds the "relatedEntityType" (e.g. "School") not a specific ID (like "School:123"). After all, the exact relationships are dynamically loaded from the data.